wdl/CallSmallVariants.wdl

```diff
@@ -0,0 +1,99 @@
+version 1.0
+
+import "Structs.wdl"
+
+task DeepVariant {
+    input {
+        File bam
+        File bai
+
+        File ref_fasta
+        File ref_fai
+
+        String model_class
+
+        String output_prefix
+
+        String? intervals
+
+        RuntimeAttr? runtime_attr_override
+    }
+
+    Int disk_size = ceil(size(bam, "GB")) + 50
+
+    String extra_args = if defined(intervals) then "--regions=~{intervals}" else " "
+
+    command <<<
+
+        # example from https://github.com/google/deepvariant/blob/r0.8/docs/deepvariant-quick-start.md
+        set -euo pipefail
+
+        # this is just to future proof ourselves for ONT data
+        if [[ ~{model_class} != "PACBIO" ]]; then
+          echo "requesting a model type not supported by the model: ~{model_class}"
```

Comment:
Good catch.  There's a different tool for calling SNPs from Nanopore data: https://www.nature.com/articles/s41467-019-09025-z .  I'll add an issue to cover implementing and evaluating this tool.

Reply:
Yeah, not sure if DV is going to support ONT data in the future, but this allows us to do an `elif` later, if that happens.